drivers/hmis/lib/hmis_util/json_forms.rb

```diff
@@ -318,7 +319,7 @@ def load_definition(form_definition:, identifier:, role:, title: nil)
 
     # Load form definitions for HUD assessments
     public def seed_assessment_form_definitions
-      roles = [:INTAKE, :EXIT, :UPDATE, :ANNUAL]
+      roles = [:INTAKE, :EXIT, :UPDATE, :ANNUAL, :POST_EXIT]
```

Comment:
This will create a "default instance" for Post-Exit assessment, meaning it sill show up for all projects regardless of funder. Unlike other HUD assessments, we don't want that to happen for post-exit assessments. Post-exit rules are going to be configured on an installation-by-installation basis most likely. So, I don't think we should run lines 345-348 if `role == :POST_EXIT`